lib/internal/test_runner/runner.js

```diff
@@ -113,7 +113,7 @@ function filterExecArgv(arg, i, arr) {
   !ArrayPrototypeSome(kFilterArgValues, (p) => arg === p || (i > 0 && arr[i - 1] === p) || StringPrototypeStartsWith(arg, `${p}=`));
 }
 
-function getRunArgs(path, { forceExit, inspectPort, testNamePatterns, only }) {
+function getRunArgs(path, { forceExit, inspectPort, testNamePatterns, testSkipPatterns, only }) {
```

Comment:
missing documentation on the new `run` option:
https://github.com/nodejs/node/blob/f73e2494ac36894469fb1c757b8103384d382b48/doc/api/test.md#L1216